Dropdown/src/Dropdown.test.js

```diff
@@ -0,0 +1,136 @@
+jest.mock("./style.less", () => jest.fn());
+jest.useFakeTimers();
+
+import React, {PropTypes} from "react";
+import Dropdown from "./Dropdown";
+import Collapse from "react-collapse";
+
+import {configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15.4';
+import toJson from 'enzyme-to-json';
+
+import {shallow, mount, render} from "enzyme";
+
+configure({adapter: new Adapter()});
+
+const options = [
+    {value: "first", label: "First Label", searchableValue: "first"},
+    {value: "second", label: "Second Label", searchableValue: "second"},
+    {value: "third", label: "Third Label", searchableValue: "third"}
+];
+
+describe("Dnn Dropdown component", () => {
+
+    it("Renders with minimal setup", () => {
+
+        const wrapper = shallow(<Dropdown options={[]} value={null} onSelect={f => f}/>);
+        expect(wrapper).toMatchSnapshot();

```

Comment:
I'm not familiar with Jest but is there a risk that `toMatchSnapshot` will create brittle tests?

Reply:
it creates final html according to current state of the application, kind of rendering. This test ensure that generated code is not changed from the previous time it was executed.

Reply:
[This article](http://randycoulman.com/blog/2016/09/06/snapshot-testing-use-with-care/) nicely puts what I had in mind:

> Another potential issue with snapshot testing is that it can make for fragile tests. For example, a test might be concerned with only a few details of the captured output, but the test will fail when **any** detail of the captured output changes.

Personally I'd use with care. It looks like powerful tool but in more complex cases may cause additional work.